pep-B.rst

```diff
@@ -156,6 +156,23 @@ Specification
     `Visualization <../install_locations.html>`_
 
 
+    **Meeting conclusion:**
+        Patches of Python, distutils or pip not needed.
+         - split binaries of system-python and normal python
+         - set sys.prefix of python to /usr/local
+         - set sys.prefix of system_python to /usr
+         - require python3 in system_python
+         - include system-python libs in sys.path of python
+         - exclude everything except system-python libs in sys.path of system-python
+         - change macro %{__python3} to system-python (py3_install, py3_build will work automaticly)
+
+        sudo pip, python setup.py install will use python. Install locations from Solution2 will be
+        used.
+
+        Problem with scripts that only require usr/bin/python3
```

Comment:
This will only affect 45 out of 2880 Python packages: http://post-office.corp.redhat.com/archives/python-maint/2016-October/msg00066.html